GirafRest.Test/Services/GirafServiceTest.cs

```diff
@@ -97,7 +108,32 @@ public void CheckProtectedOwnership_DoNotOwnResource_False()
             var res = gs.CheckProtectedOwnership((Pictogram)notUserDepartment.Resources.First().Pictogram, user);
 
             Assert.False(res.Result);
-        }
+        }
         #endregion
+        #region IpRateLimiting
+        [Fact]
+        public async Task CheckIpRateLimiting_ToManyRequests_True_StatusCode429()
+        {
+            _server = new TestServer(new WebHostBuilder().UseStartup<Startup>());
+            _client = _server.CreateClient();
+
+            HttpResponseMessage response = null;
+            for (int i = 0; i < 5; i++)
```

Comment:
A short comment could be used here, to clarify that you make 5 requests to check if you receive the 429 Response code. For easier readability.